job_bundles/vray_render/README.md

```diff
@@ -0,0 +1,13 @@
+# V-Ray sample job bundle
+Use this job bundle to create a V-Ray job to use with AWS Deadline Cloud. \
+Read more about job bundle and how to use it with AWS Deadline Cloud [here](../README.md). \
+Read more about how to customize this job bundle bellow
```

Comment:
nit: typo ``bellow``

Reply:
Will fix in the next PR along with adding more fields like mention below